planner/aamwriter/src/main/resources/examples/atos_case_study_aam.yml

```diff
@@ -73,7 +73,7 @@ groups:
         benchmark_platform: hp_cloud_services.2xl
     - dependencies:
         operation_webservices: '2'
-    - QoSRequirements:
+    - AppQoSRequirements:

```

Comment:
You are right. I will update the aams in the examples by tomorrow. Meanwhile, you check a generated aam here: https://gist.github.com/rosogon/d17a1196ca93bb58f599